Compile error on `opam install edn`

```
$ opam install edn
The following actions will be performed:
  ∗  install edn 0.1.5

=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[edn] Archive in cache

=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ERROR] The compilation of edn failed at "ocaml pkg/pkg.ml build --pinned false".

#=== ERROR while installing edn.0.1.5 =========================================#
# opam-version 1.2.2
# os           darwin
# command      ocaml pkg/pkg.ml build --pinned false
# path         /Users/will/.opam/mim/build/edn.0.1.5
# compiler     4.06.1
# exit-code    1
# env-file     /Users/will/.opam/mim/build/edn.0.1.5/edn-82903-1872ec.env
# stdout-file  /Users/will/.opam/mim/build/edn.0.1.5/edn-82903-1872ec.out
# stderr-file  /Users/will/.opam/mim/build/edn.0.1.5/edn-82903-1872ec.err
### stdout ###
# [...]
# ocamlfind ocamldep -modules src/edn_common.ml > src/edn_common.ml.depends
# ocamlfind ocamldep -modules src/edn_writer.ml > src/edn_writer.ml.depends
# ocamlfind ocamlc -c -g -bin-annot -I src -I test -o src/edn_common.cmo src/edn_common.ml
# ocamlfind ocamlc -c -g -bin-annot -I src -I test -o src/edn_writer.cmo src/edn_writer.ml
# ocamlfind ocamlc -c -g -bin-annot -I src -I test -o src/edn_json.cmo src/edn_json.ml
# + ocamlfind ocamlc -c -g -bin-annot -I src -I test -o src/edn_json.cmo src/edn_json.ml
# File "src/edn_json.ml", line 34, characters 9-31:
# Error: This expression has type bytes but an expression was expected of type
#          string
# Command exited with code 2.
### stderr ###
# pkg.ml: [ERROR] cmd ['ocamlbuild' '-use-ocamlfind' '-classic-display' '-j' '4' '-tag' 'debug'
#      '-build-dir' '_build' 'opam' 'pkg/META' 'CHANGES.md' 'LICENSE.md'
#      'README.md' 'src/edn.a' 'src/edn.cmxs' 'src/edn.cmxa' 'src/edn.cma'
#      'src/edn.cmx' 'src/edn.cmi' 'src/edn.mli' 'src/edn_common.cmx'
#      'src/edn_parser.cmx' 'src/edn_writer.cmx' 'src/edn_read.cmx'
#      'src/edn_util.cmx' 'src/edn_json.cmx' 'src/edn_cconv.a'
#      'src/edn_cconv.cmxs' 'src/edn_cconv.cmxa' 'src/edn_cconv.cma'
#      'src/edn_cconv.cmx' 'src/edn_cconv.cmi' 'src/edn_cconv.mli']: exited with 10



=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
The following actions failed
  ∗  install edn 0.1.5
No changes have been performed
```

--- a/src/edn_writer.ml
+++ b/src/edn_writer.ml
@@ -82,4 +82,4 @@ and write_char buf v =
 let to_string edn =
   let buf = Buffer.create 256 in
   write buf edn;
-  Buffer.to_bytes buf
+  Buffer.contents buf

